frontend/src/components/password/PasswordInput.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import RevealPasswordToggle from "src/components/password/RevealPasswordToggle";
+import { Input } from "src/components/ui/input";
+
+type PasswordInputProps = Omit<
+  React.InputHTMLAttributes<HTMLInputElement>,
+  "type" | "onChange"
+> & {
+  onChange?: (value: string) => void;
+};
+
+export default function PasswordInput({
+  onChange,
+  placeholder,
+  name = "password",
```

Comment:
is this field needed? what's `name` used for?

Reply:
for eg. name="confirm-password", name="unlock-password" used in same screen.

Reply:
@pavanjoshi914 I don't think the name is needed because we do not do traditional form submissions. Can you remove it and double check nothing is broken?

Reply:
removed. nothing is broken